eo.json

```diff
@@ -1327,7 +1327,7 @@
         "Tutorial.BotDesigner.ColorOptions.Secondary" : "Dua",
         "Tutorial.BotDesigner.ColorOptions.Underbody" : "Subkorpa",
         "Tutorial.BotDesigner.ColorOptions.Emission" : "Emisio",
-        "Tutorial.BotDesigner.ColorOptions.PickColor" : "Elektu Koloron",
+        "Tutorial.BotDesigner.ColorOptions.PickColor" : "Elekti Koloron",
```

Comment:
This was an addition of mine. 
This is part of the small tutorial to build your own Neos robot avatar, and in the mood of tutorials, the program generally suggests to the newcomer what to do in a procedural fashion. So, in that mood, I let it with -u to actually invite the user to try the colors and learn.

However, I don't see a problem with leaving it as an indication too, using -i. So do as you see fit. I will agree

Reply:
I agree that it should be imperative because it is telling the user to do something. I mistook it as a button to confirm a choice. Thanks for catching it!